app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
```

Comment:
csp là gì? Dòng này có tác dụng gì?

Reply:
Content Security Policy (CSP) là 1 cách hữu hiệu để giảm thiểu hoặc loại bỏ hoàn toàn các lỗ hổng Cross Site Scripting (XSS).
csp_meta_tag giúp tạo ra csp-nonce(là 1 chuỗi random), mỗi lần load trình duyệt thì nonce lại được tạo mới nên việc tấn công vào trang web sẽ trở nên khó khăn hơn